static/src/stylesheets/module/commercial/paidfor/_paidfor-content.scss

```diff
@@ -57,10 +57,12 @@
     }
 }
 
-.paidfor-box {
+.paidfor-box--header,
```

Comment:
In theory the whole thing is not using BEM. The correct way would be: .paidfor (main object) and then .paidfor__box (element of the object) and .paidfor__box--text (modification of the element).